backend/app/api/deps.py

```
@@ -38,6 +40,25 @@ def get_db() -> Generator[Session, None, None]:
+    """Get an async database session.
+    
+    Yields:
+        AsyncSession: An async SQLAlchemy session
+    """
+    async with AsyncSession(async_engine) as session:
+        yield session
+
+
+def get_storage_service() -> StorageInterface:
+    """Get the storage service implementation.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)